### ISSUE TITLE
[RFE]: Provide full schema docs for presubmit.yml

### ISSUE THREAD

**Issue by jsharpe** [bazelbuild/bazel-central-registry#2364]
### What happened?

Changes requested to presubmit.yml in PR review; schema of file / best practice has changed since last submission of module		

### Version

N/A

### Any other information?

Docs currently link to CI repo for reference for presubmit.yml but its not really documented there. Docs provide some examples but there is not a full reference of all valid keys / structure that the code can parse.
Workaround tends to be to find a similar module and copy presubmit.yml from them instead of having reference documentation to construct a file from scratch from.

**Comment by jsharpe** [comment]
Note that https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#configuring-a-pipeline is not a full specification for the format - its documentation by example; it doesn't list for instance the valid sets of keys under each section within the yaml file.

**Comment by jsharpe** [comment]
@Wyverald I don't think that PR resolves this RFE - it doesn't provide a full schema definition for the yaml file; I want a complete set of valid keys under each section documented. Please reopen this issue.

**Comment by Wyverald** [comment]
@meteorcloudy
